### PR TITLE
feat(ui): add new shortcut to show entity list

### PR DESCRIPTION
**Pull request by stonebuzz**
new shortcut to open entity form

```ctrl + alt + e```

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !23907

Depend : [11517 ](https://github.com/glpi-project/glpi/pull/11517)

### CODE REVIEW

**Review by trasher — APPROVED** [review]

File: templates/layout/parts/profile_selector.html.twig [hunk 1]
```diff
@@ -65,13 +65,16 @@
...
    <div class="dropdown dropstart">
-      <a href="#" class="dropdown-item dropdown-toggle" data-bs-toggle="dropdown" data-bs-auto-close="outside" title="{{ current_entity }}">
+      <a href="#" class="dropdown-item dropdown-toggle dropdown-list-entity" data-bs-toggle="dropdown" data-bs-auto-close="outside" title="{{ current_entity }}">
          <i class="fa-fw ti ti-stack"></i>
          {{ current_entity_short|u.truncate(35, '...') }}
       </a>
       <div class="dropdown-menu p-3">
          <h3>{{ __('Select the desired entity') }}</h3>
 
+         <div class="mb-3">
```
Comment (orthagh) [inline templates/layout/parts/profile_selector.html.twig:75]:
```suggestion
         <div class="alert alert-info" role="alert">
```

**Review by orthagh — CHANGES_REQUESTED** [review]
One small note, there is some clipping when calling the shortcut
i don't know if it's possible to avoid it 

Not blocking for merge

**Comment by stonebuzz** [comment]
> One small note, there is some clipping when calling the shortcut i don't know if it's possible to avoid it
> 
> Not blocking for merge

Let me check

**Review by orthagh — APPROVED** [review]

**Comment by stonebuzz** [comment]
I tried to give focus to the input but without success.

Boostrap implements its own listener t to know when the dropdown is displayed but does not seem to work

```javascript
    $(".dropdown-list-entity").on('shown.bs.dropdown', function(){
        setTimeout(function(){
            $(this).parent().find("input").trigger("focus");
        }, 0);

    });
````

A simple call like this does not seem to work either
```javascript
   $('input[name=entsearchtext]').trigger("focus");
````

If anyone has an idea I'm interested

**Review by trasher — APPROVED** [review]

**Comment by trasher** [comment]
@stonebuzz you have lint issue ;)

File: js/fuzzysearch.js [hunk 2]
```diff
@@ -50,6 +50,20 @@ $(function() {
...
+
+    async function open_entity_selection(e) {
+        e.stopPropagation();
+        e.preventDefault();
+        $('.avatar').trigger("click");
+        await new Promise(r => setTimeout(r, 100));
+        $('.dropdown-list-entity').dropdown('toggle');
+        $('input[name=entsearchtext]').filter(":visible")[0].focus();
+    }
+
```
Comment (cedric-anne) [inline js/fuzzysearch.js:66]: This code should not be here, as it is not related to fuzzy search. Maybe it should be moved into `templates/layout/parts/profile_selector.html.twig`, to be placed at the same place as the HTML code it controls..

**Comment by cedric-anne** [comment]
I pushed 2 commits:
1. I moved translation logic into view.
2. I added a stylesheet rule to prevent wrapping of `<kbd>` shortcuts.

But I was not able to use the feature on Chrome:
image

**Comment by cedric-anne** [comment]
> But I was not able to use the feature on Chrome: image

Fixed.

File: templates/layout/parts/profile_selector.html.twig [hunk 3]
```diff
@@ -67,20 +67,27 @@
...
+            {{ __("Tip: You can call this modal with %s keys combination")|format('<kbd>' ~ shortcut ~ '</kbd>')|raw }}
+         </div>
          <div class="alert alert-info" role="alert">
             <i class="fas fa-info-circle"></i>
             <span class="ms-2">
                {% set recursive_icon %}
                   <i class="fas fa-angle-double-down" title="{{ __('+ sub-entities') }}"></i>
                {% endset %}
-               {{ __('Click on the %s icon to see the entity and its sub-entities')|format(recursive_icon)|raw }}
+               {{ __('Click on the %s icon to load the entity and its sub-entities')|format(recursive_icon)|raw }}
```
[the remark this answers is not in the document]
Comment (AdrienClairembault) [reply templates/layout/parts/profile_selector.html.twig:90]: It was discussed with myself and @orthagh 

The 'see' label is misleading, clicking on the icon does not show you any entities, it load them.
The actual button to see the sub entities is on the left side.

[the remark this answers is not in the document]
Comment (AdrienClairembault) [reply templates/layout/parts/profile_selector.html.twig:90]: image

**Comment by orthagh** [comment]
for the last, probably fixed by #11521

**Review by AdrienClairembault — APPROVED** [review]

**Pull request merged**